### PR TITLE
Refactor the presenter

### PR DESCRIPTION
**Pull request by smashwilson**
Break `app.js` into a few smaller files. Tease apart some of the callback triangle-of-doom in the content handler.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -5,91 +5,19 @@
...
-}
-
-set_mapping_service_url();
-set_content_service_url();
-
-var request = require('request'),
-    urljoin = require('url-join'),
-    express = require('express'),
-    exphbs  = require('express-handlebars');
+config.configure(process.env);
```
Comment (kenperkins) [inline app.js:14]: Why pass `process.env` here?

Comment (smashwilson) [inline app.js:14]: So that I can add unit tests.

Comment (kenperkins) [inline app.js:14]: hrm. I hate it when you're smart.

**Comment by smashwilson** [comment]
Up and running.

**Pull request merged**